test/Modules/ModulesGraded.jl

```diff
@@ -1091,3 +1091,36 @@ end
+  I = ideal(R, minimal_generating_set(I))
+  F = graded_free_module(R, 1)
+  sub_F, inc = sub(F, [g*F[1] for g in gens(I)])
+  M = cokernel(inc)
+  A, _ = quo(R, I)
+  # To reproduce the string on the right hand side, evaluate 
+  #   `"$(oscar.minimal_betti_table(M))"` 
+  # and insert the result here; after verification of the result!
+  @test "$(oscar.minimal_betti_table(A))" == "       0  1  2  3  4  \n0    : 1  -  -  -  -  \n1    : -  -  -  -  -  \n2    : -  7  10 5  1  \n3    : -  -  -  -  -  \n4    : -  -  -  -  -  \n---------------------\ntotal: 1  7  10 5  1  \n"
+
+  @test "$(oscar.minimal_betti_table(M))" == "       0  1  2  3  4  \n0    : 1  -  -  -  -  \n1    : -  -  -  -  -  \n2    : -  7  10 5  1  \n3    : -  -  -  -  -  \n4    : -  -  -  -  -  \n---------------------\ntotal: 1  7  10 5  1  \n"
+  @test "$(oscar.minimal_betti_table(I))" == "$(oscar.minimal_betti_table(sub_F))"
+
```

Comment:
I would suggest to put one slightly non-trivial example in the doctests and only compare content here, not strings.